setup.py

```diff
@@ -85,12 +86,21 @@
     datafiles = [os.path.join('data',os.path.basename(f)) for f in datafiles]
     package_data = {'pyproj':datafiles}
 
+# store pyproj version information (from _proj.pyx) in version variable
+# (taken from Fiona)
+with open('_proj.pyx', 'r') as f:
+    for line in f:
+        if line.find("__version__") >= 0:
+            version = line.split("=")[1].strip()
+            version = version.strip('"')
+            version = version.strip("'")
```

Comment:
Thanks for reviewing.  Good suggestions.  

I must admit `version = line.split("=")[1].strip()[1:-1]` is a bit on the cryptic side.  I guess we can say that there is magic inside.  :tophat: :rabbit2:

I'll revise once jswhit comments. Thanks.